schema.graphql

```diff
@@ -419,6 +420,43 @@ type TransferManager @entity {
   updatedBlock: Block!
 }
 
+type StatType @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset!
+  opType: String!
+  claimType: String
+  claimIssuer: Identity
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferCompliance @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset! @index(unique: false)
+  type: String!
+  statType: StatType!
+  # value is present only in case of MaxInvestorCount and MaxInvestorOwnership
+  value: BigInt
```

Comment:
I think it would be nicer to have `value` as something like `BigInt | StatType`, vs having optional fields.

Basically when its a "scoped claim", the "value" is a tuple (StatClaim, issuer, min, max?), I think we are capturing this in StatType, so do we need the `claim...` fields in here as well?

Either value is present, or if claim stuff is there, then 3/4 fields are not optional

Reply:
I think the flat values will work, but add some extra confusion for the consumer

Reply:
I don't think we can provide optional types like `BigInt | StatType`. I'll keep the flat structure for now. 

To avoid confusion what we can do is rename `value` to `countValue` and `min, max` as `minClaimPercentage, maxClaimPercentage` or `minClaimBalance, maxClaimBalance` . Maybe that would help?

Reply:
I think having the names spelled out would make it clearer.

`maxClaimCount` is optional, but `maxClaimOwnership` isn't btw.

Reply:
Introducing all named variables means a lot of variables, hence i stuck with min and max here, so that consumer can deduce what it means based on other values (type, statType). I think for now it should work.

Reply:
Makes sense